devices/tuya.js

```diff
@@ -1171,4 +1171,21 @@ module.exports = [
             e.energy(), exposes.enum('power_outage_memory', ea.STATE_SET, ['on', 'off', 'restore'])
                 .withDescription('Recover state after power outage')],
     },
+    {
+        fingerprint: [{modelID: 'DoorBell-EM', manufacturerName: 'HEIMAN'},
+            {modelID: 'TS0211', manufacturerName: '_TZ1800_ladpngdx'}],
+        model: 'HS2DB',
```

Comment:
`HS2DB` is the model number of the HEIMAN device, `TS0211` fits better here.

I'm still not entirely sure if moving them together is better, especially since Lidl is a very recognisable manufacturer (imho its different from some random Chinese AliExpress brands)

Reply:
I picked HS2DB as the model, as that is what it appears to be branded as on the [Heiman website](http://www.heimantech.com/product/?type=detail&id=23) I've done what you suggested though, and changed it to TS0211.

I am not entirely sure either if moving it to the TuYa file makes sense. The original PR was for it sitting in the Heiman file which makes a lot more sense when it comes to how the whitelabling of this device has been done.

I am open to suggestions - I don't mind reverting the removal I did of the Lidl device, and using this PR to push the Heiman device separately?

Reply:
Lets push the heiman device separately indeed, I think that is less confusing for users.

Reply:
Do you mind if I use this same PR? or do you want a fresh one?

Reply:
Same pr is fine